graphql_compiler/schema_generation/sqlalchemy/schema_graph_builder.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+from collections import OrderedDict
+import warnings
+
+from graphql.type import GraphQLBoolean, GraphQLFloat, GraphQLInt, GraphQLString
+import six
+import sqlalchemy.sql.sqltypes as sqltypes
+
+from ...schema import GraphQLDate, GraphQLDateTime, GraphQLDecimal
+from ..schema_graph import PropertyDescriptor, SchemaGraph, VertexType
+
+
+# TODO(pmantica1): Add scalar mapping for the following classes: Interval, and Time.
+# We do not currently plan to add a mapping for JSON and Binary objects.
+SQL_CLASS_TO_GRAPHQL_TYPE = OrderedDict([
+    (sqltypes.String, GraphQLString),
+    (sqltypes.Integer, GraphQLInt),
```

Comment:
Are these actually the same? What's the largest `sqltypes.Integer`?

Reply:
Big int is signed 8 bytes. GraphQLInt is signed 4 bytes.

Reply:
More generally, we should look at all minor differences among the Orientdb, python, and sqlalchemy versions of the same type, and make sure we have a plan on how to cross boundaries.

Reply:
We'll probably need to (eventually) add a new custom scalar type, maybe call it `Int64` to signify it's 64-bit. For now, we can do the easy but wrong thing and use the `Int` type, so long as we open an issue so we remember to fix it later.

Reply:
We have this same issue in OrientDB.